### PR TITLE
Move to Font Awesome Icon for Lesson Extras Bubble

### PR DESCRIPTION
**Pull request by dmcavoy**
This is the final thing that needed to be moved to font awesome for the lesson/stage extras!

# Not Selected Level

## Before
Screen Shot 2019-06-03 at 2 27 15 PM

## After 

Screen Shot 2019-06-03 at 3 07 14 PM

# Hover

## Before
Screen Shot 2019-06-03 at 2 27 05 PM

## After 

Screen Shot 2019-06-03 at 3 07 19 PM

# Selected Level

## Before
Screen Shot 2019-06-03 at 2 26 55 PM

## After 

Screen Shot 2019-06-03 at 3 07 28 PM

### CODE REVIEW

File: apps/src/templates/progress/StageExtrasProgressBubble.jsx [hunk 1]
```diff
@@ -2,29 +2,34 @@ import React, {Component} from 'react';
...
   },
+  flagNormal: {
+    color: color.white
+  },
+  checkFlagNormal: {
+    color: color.lighter_gray
+  },
   focused: {
-    backgroundImage: `url('${assetUrl('media/common_images/flag_active.png')}')`
+    color: '#32CD32'
```
Comment (maddiedierker) [inline apps/src/templates/progress/StageExtrasProgressBubble.jsx]: which color is this? i'd like for it to be a constant if possible, so i'm wondering if we should:
1. add this color to our color constants, or
2. use the most similar color that is already in our color constants

File: apps/src/templates/progress/StageExtrasProgressBubble.jsx [hunk 2]
```diff
@@ -41,14 +46,29 @@ class StageExtrasProgressBubble extends Component {
       <a
         href={stageExtrasUrl + currentLocation().search}
         style={{
-          ...styles.main,
-          ...(onStageExtras && styles.focused)
+          ...styles.main
```
Comment (maddiedierker) [inline apps/src/templates/progress/StageExtrasProgressBubble.jsx]: now that there's only one style object being used here, this line can be simplified to:
```
style={styles.main}
```

Comment (maddiedierker) [reply apps/src/templates/progress/StageExtrasProgressBubble.jsx]: (same on lines 56-58)

**Review by maddiedierker — APPROVED** [review]
❤️ one checkered flag to rule them all

File: apps/src/templates/progress/StageExtrasProgressBubble.jsx (hunk 1 above)
Comment (dmcavoy) [reply apps/src/templates/progress/StageExtrasProgressBubble.jsx]: I'll add it - because every other green color was not even close

**Review by Erin007 — APPROVED** [review]

**Pull request merged**